package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "cql-exec-fhir",
-  "version": "2.0.1",
+  "version": "2.0.2",
```

Comment:
This PR only updates a dev dependency, which I don't think has any impact on consumers of the module.  So... do we really need to bump the version number for this?

Reply:
Depending upon how they're managing dependencies, the current version may get flagged when they do an audit. Bumping the patch version number provides a clear path for getting rid of these audit messages.